addons/gitlab/api.py

```diff
@@ -11,8 +11,9 @@
 from addons.gitlab.settings import DEFAULT_HOSTS
 
 # Initialize caches
-https_cache = cachecontrol.CacheControlAdapter()
-default_adapter = HTTPAdapter()
+https_cache = cachecontrol.CacheControlAdapter()  # isn't used anywhere, maybe delete?
+default_adapter = HTTPAdapter()  # isn't being used anywhere, maybe delete?
```

Comment:
Well spotted!  A lot of our addons began life as a cloned version of another addon.  This one was cloned from the github addon, which depends on (and uses) `cachecontrol` and `python-magic`.  But the gitlab addon never used those modules.  I will delete this code and the deps from gitlab's requirements.txt